policy/policy-authority-data.go

```diff
@@ -155,15 +172,10 @@ func (padb *PolicyAuthorityDatabaseImpl) allowedByWhitelist(host string) bool {
 func (padb *PolicyAuthorityDatabaseImpl) CheckHostLists(host string, requireWhitelisted bool) error {
```

Comment:
The goal here is to actually pass through the new error type to the caller so we get an InternalServer error. So I'll update the comment on CheckHostLists.